src/einsteinpy/utils/__init__.py

```diff
@@ -1,4 +1,9 @@
 from .bl_coord_transforms import *
 from .coord_transforms import *
 from .scalar_factor import scalar_factor, scalar_factor_derivative
-from .schwarzschild_utils import schwarzschild_radius, time_velocity
+
+from .schwarzschild_utils import (  # isort:skip
```

Comment:
Some final touches, as the `isort` problem is now fixed. Can you remove the `# isort:skip` from here and run `black` and `isort`